Source/WebKit/UIProcess/RemoteLayerTree/mac/RemoteLayerTreeDrawingAreaProxyMac.h

```diff
@@ -92,7 +92,7 @@ friend class RemoteScrollingCoordinatorProxyMac;
     std::optional<DisplayLinkObserverID> m_fullSpeedUpdateObserverID;
     std::unique_ptr<RemoteLayerTreeDisplayLinkClient> m_displayLinkClient;
     WebCore::PlatformLayerIdentifier m_pageScalingLayerID;
-    bool m_usesOverlayScrollbars { false };
+    std::optional<bool> m_usesOverlayScrollbars { };
```

Comment:
We try to avoid `std::optional<bool>` because it can get very confusing. It's better to write the code in a way where we can always have a value.

Reply:
Changed to use ScrollerStyle enum. I don't think there's a good way to implement this without using an optional.